ui/app/AppLayouts/Profile/panels/ContactsListPanel.qml

```diff
@@ -36,79 +37,61 @@ Item {
     signal rejectionRemoved(string publicKey)
     signal textClicked(string publicKey)
 
-    visible: contactsList.count > 0
-
     StyledText {
         id: title
         anchors.left: parent.left
         anchors.leftMargin: Style.current.padding
-        visible: contactListRoot.title !== ""
-        text: contactListRoot.title
+        visible: contactsList.count > 0 && root.title !== ""
+        text: root.title
```

Comment:
`property alias title:  titleStyledText.text` :) And pls, if you have time fix `id: title` (I see a conflict with `root.title`)